src/main/java/org/kainos/groupA/models/Competency.java

```diff
@@ -0,0 +1,34 @@
+package org.kainos.groupA.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Competency {
+    public String getComp_name() {
+        return comp_name;
+    }
+
+    public void setComp_name(String comp_name) {
+        this.comp_name = comp_name;
+    }
+
+    public String getComp_description() {
+        return comp_description;
+    }
+
+    public void setComp_description(String comp_description) {
+        this.comp_description = comp_description;
+    }
+
+    private String comp_name;
+    private String comp_description;
+
+    @JsonCreator
+    public Competency(
```

Comment:
you'll probably want to return the band name as well if you want to display it on the UI

Reply:
band name added to competency model as well as is returned from query